cli/alba.go

```diff
@@ -0,0 +1,114 @@
+package main
```

Comment:
acho que teremos vários arquivos aqui, talvez um para cada commando ou um para cada grupo de comandos relacionados. Acho que `alba.go` é um nome muito geral

Reply:
Eu pensei nela como um único arquivo e cada comando sendo adicionado na estrutura de comandos do urfave `app.Commands = []cli.Command{`, pelo menos nesse primeiro momento. Esse é um [exemplo das configurações dos comandos](https://itnext.io/how-to-create-your-own-cli-with-golang-3c50727ac608).
Onde para usar a CLI seja sempre `alba <comando> <flags ou subcomandos> ` pois penso na `alba` como o serviço que provê determinadas funcionalidades. Tipo um `apt`.
Mas talvez possamos adicionar os subcomandos em outros arquivos para ajudar na organização, e depois importamos eles na lista de comandos vinculadas ao app.

Reply:
> Eu pensei nela como um único arquivo e cada comando sendo adicionado na estrutura de comandos do urfave `app.Commands = []cli.Command{`, pelo menos nesse primeiro momento. Esse é um [exemplo das configurações dos comandos](https://itnext.io/how-to-create-your-own-cli-with-golang-3c50727ac608).
> 

O  exemplo é introdutório e tem ações simples, por isso, cabem bem num só arquivo. 

> Onde para usar a CLI seja sempre `alba <comando> <flags ou subcomandos> ` pois penso na `alba` como o serviço que provê determinadas funcionalidades. Tipo um `apt`.

Ok.

> 
> Talvez possamos adicionar os subcomandos em outros arquivos para ajudar na organização, e depois importamos eles na lista de comandos vinculadas ao app.

Não precisaremos importar, todos os arquivo dentro de um mesmo diretório estão num mesmo pacote. Você pode ter o `app.Commands = []cli.Command{' em `cli.go` e essa adicionar commandos que são declarados em arquivos diferentes. As funções nesses arquivos podem ser chamadas transparentemente de `cli.go`.

Por exemplo, supondo que `collector.go` contenha os comandos relacionados a `alba collector`, em particular, o subcomando `alba collector add --file=myfile.json`. Nesse caso podemos deixar o commando `AddCollector()` em `collector.go` e chamar `AddCollector` a partir de  `cli.go`.

Reply:
Ok